simulator/risc_v/t/unit_test.cpp

```diff
@@ -19,6 +19,43 @@ TEST_CASE("RISCV disassembly")
     CHECK( RISCVInstr<uint32>(0x30200073).get_disasm() == "mret");
     CHECK( RISCVInstr<uint32>(0x30202373).get_disasm() == "csrrs $medeleg, $t1, $zero");
     CHECK( RISCVInstr<uint32>(0x30205073).get_disasm() == "csrrwi $medeleg, $zero, 0x0");
+    CHECK( RISCVInstr<uint32>(0x00004082).get_disasm() == "c_lwsp $rs, 0x0($sp)");
```

Comment:
What is the point to have the 0000 in the beginning of each instruction?

Reply:
Just to align code structure.

Reply:
OK. IMO it's more clear to indicate that the instructions here are 16 bits long...